util/src/entities/Config.ts

```diff
@@ -49,6 +49,10 @@ export interface ConfigValue {
 		endpointPrivate: string | null;
 	};
 	general: {
+		instanceName: string;
+		instanceDescription: string | null;
+		frontPage: string | null;
```

Comment:
What is the purpose of ``frontPage``?

Reply:
It's the webpage of the instance. It can be anything such as portfolio, blog, forum, github organisation or literally any site representing the holders of the instance.

Reply:
What about an additional `tosPage` field